tools/ci_build/github/windows/eager/requirements.txt

```diff
@@ -3,3 +3,8 @@ wheel
 numpy
 typing_extensions
 torch
+cerberus
```

Comment:
`cerberus` is used for training, only. Specifically for ORTTrainer. Are we supportng this in this pipeline?

Reply:
yes this is an accident, will remove it